qiita_plugins/target_gene/tgp/pick_otus.py

```diff
@@ -103,7 +103,7 @@ def generate_artifact_info(pick_out):
     filepaths = [(path_builder('otu_table.biom'), 'biom'),
                  (path_builder('sortmerna_picked_otus'), 'directory'),
                  (glob(path_builder('log_*.txt'))[0], 'log')]
-    return [['BIOM', filepaths, True, True]]
+    return [['BIOM', filepaths]]
```

Comment:
Hmmm, so does this mean that we were implying that BIOM files were submittable to EBI and VAMPS?